src/components/MessageInput/EditMessageForm.tsx

```diff
@@ -33,18 +29,15 @@ export const EditMessageForm = <
 
   const {
     clearEditingState,
-    closeEmojiPicker,
-    emojiPickerIsOpen,
```

Comment:
Removal should be reflected in the docs: closeEmojiPicker, emojiPickerIsOpen, openEmojiPicker